examples/nextjs/pages/index.tsx

```diff
@@ -0,0 +1,72 @@
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import Image from 'next/image'
+import styles from '../styles/Home.module.css'
+
+const Home: NextPage = () => {
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>Create Next App</title>
```

Comment:
Could we avoid adding these helper messages and styles to keep the example simple? Thanks.

Reply:
Done. Also removed API route as it isn't used.